pytests/repo/setup-repo.sh

```diff
@@ -80,15 +80,15 @@ repodir=${TEST_REPO_DIR}/yum.repos.d
 cachedir=${TEST_REPO_DIR}/cache/tdnf
 EOF
 
-cat << EOF > ${PUBLISH_PATH}/repodata/photon.metalink
+cat << EOF > ${PUBLISH_PATH}/metalink
 <?xml version="1.0" encoding="utf-8"?>
 <metalink version="3.0" xmlns="http://www.metalinker.org/" type="dynamic" pubdate="Wed, 05 Feb 2020 08:14:56 GMT">
  <files>
   <file name="repomd.xml">
    <verification>
    </verification>
    <resources maxconnections="1">
-    <url protocol="ftp" type="ftp" location="IN" preference="100">file://${PUBLISH_PATH}</url>
+    <url protocol="ftp" type="ftp" location="IN" preference="100">file://${PUBLISH_PATH}/repodata/repomd.xml</url>
```

Comment:
shouldn't the url protocol and type be "file" instead of "ftp"?

Reply:
Done.